libraries/lts/include/mcrl2/lts/detail/liblts_bisim.h

```diff
@@ -262,6 +266,25 @@ class bisim_partitioner
     std::vector< block_index_type > to_be_processed;
     std::vector< block_index_type > BL;
 
+    // Counter example information based on "Computing Distinguishing Formulas for Branching Bisimulation", 1991 by Henri Korver
+    // Given a block B that is split using action alpha and block B' into blocks L and R, with P as the partition just before this split, then
+    // right_child[B] = R
+    std::map < block_index_type, block_index_type > right_child;
+    // split_by_action[B] = alpha,
+    std::map < block_index_type, label_type > split_by_action;
+    // split_by_block[B] = B',
+    std::map < block_index_type, block_index_type > split_by_block;
+    // r_alpha[R] = { C \in P | \exists_{s \in R, s' \in C} : s -a-> s'} and
+    std::map < block_index_type, std::set < block_index_type > > r_alpha;
+    // r_tau[R] = { C \in P | \exists_{s \in R, s' \in C} : s -tau-> s' && C != B}.
+    std::map < block_index_type, std::set < block_index_type > > r_tauP;
+    // map from source state and action to target state, makes generating counterexample info easier
+    outgoing_transitions_per_state_action_t outgoing_transitions;
+
+    // A counter for creating fresh variables
+    int freshVarCounter = 0;
```

Comment:
For consistency with the rest of the file it might make sense to use snake case, i.e., fresh_var_counter.